src/crypto/hash.h

```diff
@@ -59,10 +58,7 @@ namespace fmt
     template <typename FormatContext>
     auto format(const crypto::Sha256Hash& p, FormatContext& ctx)
     {
-      return format_to(
-        ctx.out(),
-        "<sha256 {:02x}>",
-        fmt::join(p.h, ""));
+      return format_to(ctx.out(), "<sha256 {:02x}>", fmt::join(p.h, ""));
```

Comment:
Slightly surprised by this reformatting, are you using the right clang format?

Reply:
Yes, I think I ran the check-format.sh file with the formatting option on, but I can always check again